<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutAdmin</class>
 <widget class="QDialog" name="ajoutAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>321</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>331</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom de l'hotel:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomHotelEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent; 
    border: none; 
    border-bottom: 1px solid black; 
    color: black;  
    padding: 5px; 
    font-size: 14px;                  
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Localisation de l'hotel:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="localisationHotelEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent; 
    border: none; 
    border-bottom: 1px solid black; 
    color: black;  
    padding: 5px; 
    font-size: 14px;                  
}</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre d'etoiles:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="nbEtoilesEdit">
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: transparent; 
    border: none; 
    border-bottom: 1px solid black; 
    color: black;  
    padding: 5px; 
    font-size: 14px;                  
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="CancelHotelBtn">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(0, 0, 0); 
border-radius: 10px;
padding: 4px;
</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="conformHotelBtn">
      <property name="font">
       <font>
        <family>Helvetica CE 35 Thin</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(143, 160, 255); 
border-radius: 10px;
padding: 4px;
background-color: rgb(158, 168, 255);
</string>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica CE 35 Thin</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(99, 159, 255);</string>
   </property>
   <property name="text">
    <string>Ajoutez un hotel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
